tests/tests/namespace/test-api-rules.spec.js

```diff
@@ -145,4 +146,9 @@ context('Test API Rules in the Function details view', () => {
       .contains('Allow')
       .should('exist');
   });
+
+  it('Inspect list and delete', () => {
+    cy.navigateTo('Discovery and Network', 'API Rules');
+    deleteFromGenericList(API_RULE_NAME);
```

Comment:
I don't think it's required to delete it, since it will be deleted together with a function (I believe)